serde-api/src/main/java/io/kafbat/ui/serde/api/DeserializeResult.java

```diff
@@ -6,21 +6,19 @@
 
 /**
  * Result of {@code Deserializer} work.
+ *
+ * @param result nullable
  */
-public final class DeserializeResult {
+public record DeserializeResult(String result, io.kafbat.ui.serde.api.DeserializeResult.Type type,
```

Comment:
`io.kafbat.ui.serde.api.DeserializeResult.Type` - can we import this?
`String result` - `@nullable` won't hurt I guess considering the comment we had on line 16